src/main/java/com/cosmian/jna/findex/Leb128Serializer.java

```diff
@@ -0,0 +1,85 @@
+package com.cosmian.jna.findex;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+import com.android.dex.Leb128;
+import com.android.dex.util.ByteArrayByteInput;
+import com.android.dx.util.ByteArrayAnnotatedOutput;
+
+public class Leb128Serializer {
+
+    public static List<byte[]> deserialize(byte[] serializedUids) {
```

Comment:
In python, you've done something that I like :
You have 3 methods : deser_list, deser_map and one generic deserialize method who trigger the right deserialization type. 
I think there might be a Java thing here that I don't precisely get, but, maybe we can do something more explicite and symmetric between deserialization and serialization ?
deserialize_list / deserialize_map
serialize_list/serialize_map
?